src/asm2wasm.h

```diff
@@ -565,6 +649,16 @@ class Asm2WasmBuilder {
   }
 
   Function* processFunction(Ref ast);
+
+public:
+  CallImport* isDebugInfo(Expression* curr) {
```

Comment:
Good point, yeah. Changing to `checkDebugInfo` which is consistent with other stuff in wasm.h.